src/UnitTests.Shared/RunnerUtilities.cs

```diff
@@ -140,16 +140,17 @@ private static void AdjustForShellExecution(ref string pathToExecutable, ref str
                 p.BeginErrorReadLine();
                 p.StandardInput.Dispose();
 
+                TimeSpan timeout = TimeSpan.FromMilliseconds(timeoutMilliseconds);
                 if (Traits.Instance.DebugUnitTests)
                 {
                     p.WaitForExit();
                 }
-                else if (!p.WaitForExit(timeoutMilliseconds))
+                else if (!p.WaitForExit(timeout))
```

Comment:
Not able to wait for a `TimeSpan` and it is failing the build.

```suggestion
                else if (!p.WaitForExit(timeoutMilliseconds))
```